hw2/css/style.css

```diff
@@ -0,0 +1,130 @@
+body {
+	background-color: #f9f9f9;
+	font-size: 13px;
+	font-family: OpenSans, Ubuntu, Antic, sans-serif;
+} 
+.wrapper {
+	margin: auto;
+	padding: 50px 48px 60px 46px;
+	width: 1060px;
+	background-color: #ffffff;
+	box-shadow: 0 5px 5px rgba(0, 0, 0, 0.3);
+}
+/*HEADER*/
+.top-menu {
+	display: flex;
+	background-color: #5e5e5e;
+	background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%); 
+	margin-top: 40px;
+}
+.top-menu_item {
+	display: block;
+	padding: 14px 25px 11px;
+	border-left: 1px solid rgba(255,255,255,0);
+	border-right: 1px solid rgba(255,255,255,0);
+	text-shadow: 1px 1px 1px rgba(0,0,0,0.5);
+	color: #ffffff;
+	font-size: 15px;
+}
+.top-menu_item:hover {
+	background-image: linear-gradient(to top, #2e2e2e, #4a4a4a)	;
+    border-left: 1px solid rgba(255,255,255,1);
+    border-right: 1px solid rgba(255,255,255,1);
+}
+/*CONTAINER*/
+.container {
+	padding-top: 40px;
+	padding-bottom: 60px;
+}
+.container::after {
+	content: '';
+	display: block;
+	clear: both;
+}
+.left-menu {
+	float: left;
+	width: 240px;
+}
+.left-menu_item {
+	border: 1px solid #c8c8c8;
+	border-top: none;
+}
+.left-menu_item:first-child {
+	border-top: 1px solid #c8c8c8;
+}
+.left-menu_link {
+	display:block;
+	color: #656363;
+	padding: 15px 0px 11px 22px;
+}
+.left-menu_item:hover {
+	background-color: #f9f9f9;
+}
+.left-menu_link:hover {
+	color: #2c2c2c;
+	position: relative;
```

Comment:
:+1: now try to use transform. I've recently read that it suits better for animations